### ISSUE TITLE
Cystic lung disease

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4594]
This term label is suboptimal. 
https://hpo.jax.org/app/browse/term/HP:0005948
The term is used to directly annotate only one disease
```
$ grep 0005948 phenotype.hpoa 
OMIM:219600	CYSTIC DISEASE OF LUNG		HP:0005948	OMIM:219600	TAS					P	HPO:skoehler[2009-02-17]
```
This disease entry is legacy and is poorly defined. The phenotype we are trying to capture is pulmonary cyst. Can we relabel/improve the definitions.

**Comment by pnrobinson** [comment]
I think we can relabel to Lung cyst, syn pulmonary cyst

**Comment by pnrobinson** [comment]
Actually, I have restructured a bit and relabeled this term 'Multiple pulmonary cysts" and left 'Cystic lung disease' as an exact synonym.
